lib/Robo/Plugin/Commands/CleanCacheCommands.php

```diff
@@ -14,48 +18,70 @@
 
 class CleanCacheCommands extends \Robo\Tasks
 {
-    
+    use CliRunnerTrait;
+
     /**
      * Clean 'cache/' directory
 
+     * @param array $opts
      * @throws \RuntimeException
-     * @return nothing
+     * @return void
+     *
+     * @command cache:clean
+     * @aliases clean:cache
+     * @option  force Force clean directories without confirmation
      */
-    public function cleanCache()
+    public function cleanCache($opts = ['force' => false])
     {
+        global $sugar_config;
+        $this->bootstrap();
+        $cacheDir = isset($sugar_config) && isset($sugar_config['cache_dir']) ? $sugar_config['cache_dir'] : 'cache';
+
         $toDelete = array();
         $doNotDelete = array('Emails', 'emails', '.', '..');
-        $cacheToDelete = array('cache/Relationships',
-                               'cache/csv',
-                               'cache/dashlets',
-                               'cache/diagnostics',
-                               'cache/dynamic_fields',
-                               'cache/feeds',
-                               'cache/import',
-                               'cache/include/javascript',
-                               'cache/jsLanguage',
-                               'cache/pdf',
-                               'cache/themes',
-                               'cache/xml',
+        $cachesToDelete = array('Relationships',
```

Comment:
Place 'Relationships', on a newline, For example:
$cachesToDelete = array(
    'Relationships',
    'csv',
    ...
    ...
);

Reply:
Done.